<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_course</class>
 <widget class="QDialog" name="add_course">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1,1,1,1,1">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>添加课程信息</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_1" stretch="1,9">
     <item>
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>班级号:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_classid">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>请输入班级号(入学年份+专业代号+班级代号):如20130201</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4,1,1,2,2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>课程名称:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_course_name">
       <property name="placeholderText">
        <string>请输入课程名称</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>学期:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbox_term">
       <item>
        <property name="text">
         <string>大一上</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>大一下</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>大二上</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>大二下</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>大三上</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>大三下</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>大四上</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>大四下</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>研一上</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>研一下</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>研二上</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>研二下</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>研三上</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>研三下</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4,1,1,4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>教师:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_teacher">
       <property name="placeholderText">
        <string>请输入教师姓名</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>教室:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_classrom">
       <property name="placeholderText">
        <string>请输入教室</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4,1,1,2,2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>学时:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_period">
       <property name="placeholderText">
        <string>请输入学时</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>学分:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_credit">
       <property name="placeholderText">
        <string>请输入学分</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,1,1,1,2">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_confirm">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
